template/runner/graph_classification_iamdb/graph_classification_iamdb_CL_arguments.py

```diff
@@ -0,0 +1,46 @@
+# Utils
+import argparse
+
+# Torch
+import torch
+
+# Gale
+import models
+from template.runner.base.base_CL_arguments import BaseCLArguments
+
+
+class GraphClassificationIamdbCLArguments(BaseCLArguments):
```

Comment:
In image classification this file is called just: `CL_arguments` maybe we could rename it to be consistent

Reply:
Will be renamed in the second push.